examples/watch/Program.cs

```diff
@@ -13,8 +13,8 @@ private static void Main(string[] args)
 
             IKubernetes client = new Kubernetes(config);
 
-            var podlistResp = client.ListNamespacedPodWithHttpMessagesAsync("default", watch: true).Result;
-            using (podlistResp.Watch<V1Pod>((type, item) =>
+            var podlistResp = client.ListNamespacedPodWithHttpMessagesAsync("tester", watch: true);
```

Comment:
Keep this as "default" I changed it for my cluster, but I should have reverted it...